src/rootly_mcp_server/server.py

```diff
@@ -222,7 +217,7 @@ def create_rootly_mcp_server(
 
     # Add some custom tools for enhanced functionality
     @mcp.tool()
-    def list_endpoints() -> str:
+    def list_endpoints() -> list:
```

Comment:
Should return a list, not a string or itll look like this `"path\": \"/v1/alerts\",\n    \"method\": \"GET\",\n
  \"summary\": \"List alerts\",\n    \"description\": \"List
  alerts\"\n  },\n  {\n    \"path\": \"/v1/environments\",\n
     \"method\": \"POST\",\n    \"summary\": \"Creates an`